python-hawkular-client.spec

```diff
@@ -0,0 +1,114 @@
+%global srcname hawkular-client
+%global _docdir_fmt %{name}
+
+%if 0%{?fedora}
+%bcond_without python3
+%else
+%bcond_with python3
+%endif
+
+Name:           python-%{srcname}
+Version:        0.5.1
+Release:        1%{?dist}
+Summary:        Python client to communicate with Hawkular server over HTTP(S)
+
+License:        ASL 2.0
+URL:            https://github.com/google/%{srcname}
+Source0:        https://pypi.python.org/packages/source/h/%{srcname}/%{srcname}-%{version}.tar.gz
+
+BuildArch:      noarch
+BuildRequires:  python2-devel
+BuildRequires:  python-setuptools
+## For tests
+#BuildRequires:  python-mox
+#Requires:       python-dateutil
+#Requires:       python-gflags
+#Requires:       pytz
+
+%description
+Python client to access Hawkular-Metrics, an abstraction to invoke REST-methods 
+on the server endpoint using urllib2. No external dependencies.
+
+
+%if %{with python3}
+%package -n python3-%{srcname}
+Summary:        Python 3 client to communicate with Hawkular server over HTTP(S)
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python-tools
+# For tests
+# python-mox doesn't work with python3
+# https://bugzilla.redhat.com/show_bug.cgi?id=1209203
+#BuildRequires:  python3-mox
+Requires:       python3-dateutil
+Requires:       python3-gflags
+Requires:       python3-pytz
+
+%description -n python3-%{srcname}
+Python client to access Hawkular-Metrics, an abstraction to invoke REST-methods 
+on the server endpoint using urllib2. No external dependencies.
+%endif # with python3
+
+%prep
+%setup -qc
+mv %{srcname}-%{version} python2
```

Comment:
@yaacov @zgalor correct, this is the issue that is generating the failure at the moment, can you update this ASAP. Thanks.